app/uk/gov/hmrc/plasticpackagingtax/registration/repositories/UserDataRepository.scala

```diff
@@ -69,3 +75,17 @@ class UserDataRepository @Inject() (
     delete[A](id)(DataKey(key))
 
 }
+
+object CacheItemFormats {
+
+  val format: Format[CacheItem] = {
+    implicit val dtf: Format[Instant] = MongoJavatimeFormats.instantFormat
+    ((__ \ "_id").format[String]
+      ~ (__ \ "data").format[JsObject]
+      ~ (__ \ "modifiedDetails" \ "createdAt").format[Instant]
+      ~ (__ \ "modifiedDetails" \ "lastUpdated").format[Instant])(CacheItem.apply,
+                                                                  unlift(CacheItem.unapply)
+    )
+  }
+
```

Comment:
Is this required? I see it's a copy of `MongoCacheRepository` in the file: `MongoCacheRepository`.

Reply:
Yeah without it the Json.toJson would not work, it complains of implicit write for cacheItem